### ISSUE TITLE
Colordict dictionary 

### ISSUE THREAD

**Issue by NSTAdventure** [scubajeff/Setter#8]
1. This app contain ads and non free proprietary (gms) also this app is depending on gms how you can say they respecting user privacy? 🗿🤦

**Comment by scubajeff** [comment]
By that I mean it doesn't depend on web service to function, dictionaries are files sit in your phone, no server would know what words you are looking up. As for ads, ColorDict has not been updated for years, it's ads link is quite easy to, might already been, blocked by common ad blocker. GMS? I use it in a ROM with no GMS  and it runs perfectly fine. 

Of couse, you have every single right to challenge me on this statement. And I feel sorry if you found yourself being cheated. However, no matter what, it's your choice to use it or drop it.

**Comment by NSTAdventure** [comment]
> By that I mean it doesn't depend on web service to function, dictionaries are files sit in your phone, no server would know what words you are looking up. As for ads, ColorDict has not been updated for years, it's ads link is quite easy to, might already been, blocked by common ad blocker. GMS? I use it in a ROM with no GMS and it runs perfectly fine.
> 
> Of couse, you have every single right to challenge me on this statement. And I feel sorry if you found yourself being cheated. However, no matter what, it's your choice to use it or drop it.

why you recommend non free proprietary software inside setter app? There is one foss app that i know but idk can fully replace google translate or not:
https://f-droid.org/packages/com.sbeve.jada

You can replace those recommendation if you want to ^_°

**Comment by scubajeff** [comment]
App not found.

**Comment by NSTAdventure** [comment]
> App not found.

Huh f droid are getting weird and worst at the same time i found this app from f droid app but not found on their website???? 🥴🥴🥴🥴

Yeah me have same experience 🤦🤦🤦🤦🤦🤦🤦

**Comment by odnar-dev** [comment]
@NSTAdventure 
> Huh f droid are getting weird and worst at the same time i found this app from f droid app but not found on their website????

not available on the official f-droid repo yet
found it on izzy repo https://apt.izzysoft.de/fdroid/index/apk/com.sbeve.jada

**Comment by NSTAdventure** [comment]
> @NSTAdventure
> 
> > Huh f droid are getting weird and worst at the same time i found this app from f droid app but not found on their website????
> 
> not available on the official f-droid repo yet
> found it on izzy repo https://apt.izzysoft.de/fdroid/index/apk/com.sbeve.jada

hmm? 😳
Did i add repo without i am realizing it? 😳
Uh okay....

**Comment by scubajeff** [comment]
one of the main the reason i choose colordict is offline feature. online api can track your look up history no matter how free and open they sound. i believe it's much more worst than ad.

**Comment by NSTAdventure** [comment]
> one of the main the reason i choose colordict is offline feature. online api can track your look up history no matter how free and open they sound. i believe it's much more worst than ad

Uh maybe other FOSS dictionary? That are just a example :-/

**Comment by scubajeff** [comment]
yes. offline dictionary is the key feature requirement, another better to have is the popup window mode, which save the hassle of app switching. i make this app mainly for my own use, what i try to do is to mimic the working environment of desktop

**Comment by odnar-dev** [comment]
if i remember right, i think it work with [Aard 2](https://f-droid.org/en/packages/itkach.aard2/) out of the box.

**Comment by TPS** [comment]
> @NSTAdventure 
> > Huh f droid are getting weird and worst at the same time i found this app from f droid app but not found on their website????
> 
> not available on the official f-droid repo yet
> found it on izzy repo https://apt.izzysoft.de/fdroid/index/apk/com.sbeve.jada

Official source repo @ https://github.com/sbeve72/JADA

**Comment by yephny** [comment]
Is there really no decent libre English dictionary that works offline?

**Comment by scubajeff** [comment]
There are some, but none of them provide an Intent which other apps like Setter can call.

**Comment by yephny** [comment]
@scubajeff Sounds like a good feature request to make for those apps, then.
